sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCRelation.scala

```diff
@@ -37,6 +37,9 @@ private[sql] case class JDBCPartitioningInfo(
     numPartitions: Int)
 
 private[sql] object JDBCRelation {
+
+  import scala.collection.JavaConverters._
```

Comment:
OK, I'll merge after #10519 is closed, right now I see some changes that pretty much did what I did